vmm/src/config.rs

```diff
@@ -1959,6 +2066,14 @@ impl VmConfig {
             return Err(ValidationError::CpusMaxLowerThanBoot);
         }
 
+        if let Some(rate_limit_groups) = &self.rate_limit_groups {
+            for rate_limit_group in rate_limit_groups {
+                rate_limit_group.validate(self)?;
+
+                Self::validate_identifier(&mut id_list, &Some(rate_limit_group.id.clone()))?;
+            }
+        }
+
```

Comment:
We should also cross check `self.rate_limit_groups` with the list of `rate_limit_group` specified from `self.disks` to:
1. Report error if any `rate_limit_group` specified from `self.disks` is not defined from `self.rate_limit_groups;`
2. Drop any members from `self.rate_limit_groups` if they are not used by any `self.disks` (with warnings if you like);

Reply:
Good point. Added a check and a unit test for reporting error if any `rate_limit_group` specified from `self.disks` is not defined from `self.rate_limit_groups`.